[**@sitecore-marketplace-sdk/xmc**](../README.md)

***

[@sitecore-marketplace-sdk/xmc](../README.md) / CreateHostErrors

# Type Alias: CreateHostErrors

> **CreateHostErrors** = `object`

Defined in: [packages/xmc/src/client-xmapp/types.gen.ts:2963](https://github.com/Sitecore/sitecore-marketplace-sdk/blob/e87783cce9f115393973a45e109d17b99bf1df7e/packages/xmc/src/client-xmapp/types.gen.ts#L2963)

## Properties

### 400

> **400**: [`ProblemDetails`](ProblemDetails.md)

Defined in: [packages/xmc/src/client-xmapp/types.gen.ts:2967](https://github.com/Sitecore/sitecore-marketplace-sdk/blob/e87783cce9f115393973a45e109d17b99bf1df7e/packages/xmc/src/client-xmapp/types.gen.ts#L2967)

One or more validation errors occurred

***

### 409

> **409**: [`ProblemDetails`](ProblemDetails.md)

Defined in: [packages/xmc/src/client-xmapp/types.gen.ts:2971](https://github.com/Sitecore/sitecore-marketplace-sdk/blob/e87783cce9f115393973a45e109d17b99bf1df7e/packages/xmc/src/client-xmapp/types.gen.ts#L2971)

The host name is already in use
